sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/training/FormTrainingAsyncClient.java

```diff
@@ -356,6 +472,88 @@ public FormRecognizerServiceVersion getServiceVersion() {
+                            CustomFormModelStatus.fromString(copyOperationResult.getStatus().toString()),
+                            copyOperationResult.getCreatedDateTime(),
+                            copyOperationResult.getLastUpdatedDateTime());
+                    });
+            } catch (RuntimeException ex) {
+                return monoError(logger, ex);
+            }
+        };
+    }
+
+    private Function<PollingContext<OperationResult>, Mono<PollResponse<OperationResult>>>
+        createCopyPollOperation(String modelId) {
+        return (pollingContext) -> {
+            try {
+                PollResponse<OperationResult> operationResultPollResponse = pollingContext.getLatestResponse();
+                UUID targetId = UUID.fromString(operationResultPollResponse.getValue().getResultId());
+                return service.getCustomModelCopyResultAsync(UUID.fromString(modelId), targetId)
+                    .flatMap(modelSimpleResponse ->
+                        processCopyModelResponse(modelSimpleResponse, operationResultPollResponse));
+            } catch (HttpResponseException e) {
+                logger.logExceptionAsError(e);
+                return Mono.just(new PollResponse<>(LongRunningOperationStatus.FAILED, null));
+            }
+        };
+    }
+
+    private Function<PollingContext<OperationResult>, Mono<OperationResult>> getCopyActivationOperation(
+        String modelId, CopyAuthorization copyAuthorization) {
+        return (pollingContext) -> {
+            try {
+                Objects.requireNonNull(modelId, "'modelId' cannot be null.");
+                Objects.requireNonNull(copyAuthorization, "'copyAuthorization' cannot be null.");
+                CopyRequest copyRequest = new CopyRequest()
+                    .setTargetResourceId(copyAuthorization.getResourceId())
+                    .setTargetResourceRegion(copyAuthorization.getResourceRegion())
+                    .setCopyAuthorization(new CopyAuthorizationResult()
+                        .setModelId(copyAuthorization.getModelId())
+                        .setAccessToken(copyAuthorization.getAccessToken())
+                        .setExpirationDateTimeTicks(copyAuthorization.getExpirationDateTimeTicks()));
+                return service.copyCustomModelWithResponseAsync(UUID.fromString(modelId), copyRequest)
+                    .map(response ->
+                        new OperationResult(parseModelId(response.getDeserializedHeaders().getOperationLocation())));
+            } catch (RuntimeException ex) {
+                return monoError(logger, ex);
+            }
+        };
+    }
+
+    private static Mono<PollResponse<OperationResult>> processCopyModelResponse(
+        CopyOperationResult copyModel,
+        PollResponse<OperationResult> copyModelOperationResponse) {
+        LongRunningOperationStatus status;
+        switch (copyModel.getStatus()) {
+            case NOT_STARTED:
```

Comment:
why is there an empty case

Reply:
It is kind of a fall-through, so it will behave similarly as the next case.